Handling Military Aircraft (#599)
vatpac-technology/sops: docs/controller-skills/military.md

    **ARA**: "MAPL01 and DRGN50, MARSA terminated"

## NOCOM Operations

> **Non-Continuous Communication** (NOCOM) Procedures are when military aircraft discontinue two-way radio communications with Civil ATC. NOCOM Procedures state that ATC Approval is required when military aircraft are in controlled airspace. military aircraft under NOCOM procedures fall under multiple categories:

Reviewer: ```suggestion
**Non-Continuous Communication** (NOCOM) procedures allow military aircraft to discontinue two-way radio communications with civil ATC. ATC approval is required when military aircraft are in controlled airspace. Military aircraft under NOCOM procedures fall under multiple categories:
```

Stereo Approach charts can be found in the [RAAF TERMA](https://ais-af.airforce.gov.au/australian-aip){target=new}

## Aerodrome Responsbilities
If a controller is handling a military aircraft, they must confirm the undercarrige is down and locked when issuing a clearance to land.

!!! phraseology
    **ESA**: "VIPR42, cleared to land runway 27, check wheels"  
    **VIPR42**" "Cleared to land runway 27, 3 greens"

!!! note

>     Some aircraft may respond with a beep to indicate three greens.

Reviewer: ```suggestion
    Some aircraft may respond with a beep tone to indicate three greens. In the real world, this tone is produced by the aircraft itself, however some online pilots may choose to replicate this produce using other methods.
```
